hq_superset/views.py

```diff
@@ -104,6 +107,13 @@ class CCHQApiException(Exception):
     pass
 
 
+def convert_to_array(string_array):
```

Comment:
Might be nice to show what `string_array` is expected to look like. I love docstrings for doing this, but English also works.
```suggestion
def convert_to_array(string_array):
    """
    Converts the string representation of a list to a list.

    >>> convert_to_array("['hello', 'world']")
    ['hello', 'world']

    """
```

You could add unit tests to test the rest of the behavior ... or you could just add a few more doctests ...
```suggestion
def convert_to_array(string_array):
    """
    Converts the string representation of a list to a list.

    >>> convert_to_array("['hello', 'world']")
    ['hello', 'world']
    >>> convert_to_array("hello, world")
    Traceback (most recent call last):
      ...
    ValueError: malformed node or string: ...
    >>> convert_to_array("[None]")
    []
    >>> convert_to_array("'hello', 'world'")
    ('hello', 'world')

    """
```
I prefer to keep doctests short, and use unit tests to verify behavior, but I guess it depends where you find the balance between code readability and typing unit tests.

... Also, should we be catching that `ValueError`?

Reply:
Doctests are a great idea. Also, looking back at this function now I realize it's written a little narrow (ie. no error handling or so). I'll make some updates.